compose-app-handson: Adding Tutorial (#217)
foundriesio/docs: source/tutorials/compose-app/compose-app-file-structure.rst

File Structure
^^^^^^^^^^^^^^

The file structure in the ``containers.git`` repository is extremely important. 
It will define when the FoundriesFactory CI creates a Docker Image or/and 
creates a Docker Compose Application.

Most Docker Compose Apps structure will be similar to the ``shellhttpd`` example provided by 
default in your ``containers.git`` when you create your Factory. The same 
example you have been using in all previous tutorials.

Let’s recap the ``shellhttpd`` example structure:

.. prompt:: text

     └── containers
         └── shellhttpd
             ├── docker-build.conf
             ├── docker-compose.yml
             ├── Dockerfile
             ├── httpd.sh
             └── shellhttpd.conf

> What is important in the example is the ``shellhttpd`` directory containing 
> a ``docker-compose.yml`` or a ``Dockerfile`` file.
> 
> This is an example where the Docker Container Image and Docker Compose App are 
> present in the same folder.

Reviewer: ```suggestion
What is important in the example is the ``shellhttpd`` directory containing 
a ``docker-compose.yml`` and a ``Dockerfile`` file. CI will produce both a Docker Container image and Compose app named ``shellhttpd``
```

> present in the same folder.

Reviewer 2: @munoz0raul this one is missing

> present in the same folder.

Author: I accepted Andy's suggestion. It is in the same sentence. 

```
What is important in the example is the ``shellhttpd`` directory containing 
a ``docker-compose.yml`` and a ``Dockerfile`` file. CI will produce both a Docker Container image and Compose app named ``shellhttpd`
```
